core-services/release-controller/_repos/ocp-4.19-rhel96.repo

```diff
@@ -46,6 +46,18 @@ password_file = /tmp/mirror-enterprise-basic-auth/password
 failovermethod = priority
 skip_if_unavailable = true
 
+[rhel-96-highavailability]
+name = rhel-96-highavailability
```

Comment:
```suggestion
[rhel-9.6-highavailability]
name = rhel-9.6-highavailability
```

Reply:
whops :) too much copy paste... fixed now.

thanks for the feedback.